carservice/views.py

```diff
@@ -83,6 +84,51 @@ def post(self, request):
             return redirect('/car/read/')
         return render(request, 'car_delete.html', {'form': form})
 
+@login_required
+def carsearch(request):
+    search = request.GET.get('search')
+    cars = Car.objects.none()
+    offers = Offer.objects.none()
+    message = ''
```

Comment:
generalnie to powinno być zrobione w endpoincie, gdzie user będzie przeglądał oferty? Nie było już takiego?